.gitignore

```diff
@@ -4,3 +4,4 @@ lib
 node_modules
 npm-debug.log
 test/output
+.idea
```

Comment:
Can we please revert this. You should ignore this folder globally on your machine.
https://help.github.com/articles/ignoring-files/#create-a-global-gitignore